modules/client/examples/01-client/01-create-dao.ts

```diff
@@ -0,0 +1,58 @@
+/* MARKDOWN
+### Create a DAO
+
+The `createDao` function allows you to create a DAO using the parameters you set for it.
+*/
+
+import {
+  Client,
+  CreateDaoParams,
+  DaoCreationSteps,
+  DaoMetadata,
+  GasFeeEstimation
+} from "@aragon/sdk-client";
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiate the general purpose client from the Aragon OSx SDK context.
+const client: Client = new Client(context);
+
+const metadata: DaoMetadata = {
+  name: "My DAO",
+  description: "This is a description",
+  avatar: "image-url",
+  links: [{
+    name: "Web site",
+    url: "https://..."
+  }]
+};
+
+// Through pinning the metadata in IPFS, we can get the IPFS URI. You can read more about it here: https://docs.ipfs.tech/how-to/pin-files/
+const metadataUri = await client.methods.pinMetadata(metadata);
+
+const createDaoParams: CreateDaoParams = {
+  metadataUri,
+  ensSubdomain: "my-org", // my-org.dao.eth
+  plugins: [] // may be left as an empty array, but we recommend adding at least 1 governance plugin upon creation.
+};
+
+// Estimate how much gas the transaction will cost.
+const estimatedGas: GasFeeEstimation = await client.estimation.createDao(createDaoParams);
```

Comment:
I'm not sure but maybe we should make clear that this is step is totally optional